src/components/input/Input.js

```diff
@@ -165,20 +165,71 @@ Input.propTypes = {
     'hidden'
   ]),
 
+  /**
+   * The value of the Input
+   */
   value: PropTypes.string,
+
+  /**
+   * The initial size of the control. This value is in pixels unless the value
+   * of the type attribute is text or password, in which case it is an integer
+   * number of characters. Starting in, this attribute applies only when the
```

Comment:
That sentence is a bit strange. Starting in what?

Reply:
Whoops, my bad. I copied most of these from the [dash-core-components Input](https://github.com/plotly/dash-core-components/blob/master/src/components/Input.react.js) and didn't check them that carefully. You're right this doesn't make a whole lot of sense.